### DISCUSSION TITLE
Task exited with return code Negsignal.SIGSEGV

### DISCUSSION THREAD

**Discussion by nc-tannerum** [apache/airflow#27780] [Q&A]
Hi,
I am doing a POC on Airflow. I built my first airflow dag and it keeps failing with the message
Task exited with the return code Negsignal.SIGSEGV

I am using the python version directly (no dockers at this point in time).
I am using Python 3.9.
On macOS.

Here sample log file

> [2022-11-18T08:33:38.380-0600] {taskinstance.py:1590} INFO - Exporting the following env vars:
> AIRFLOW_CTX_DAG_OWNER=DW-Fivetran
> AIRFLOW_CTX_DAG_ID=MoneyPool_sync
> AIRFLOW_CTX_TASK_ID=Fivetran-Trigger_Sync
> AIRFLOW_CTX_EXECUTION_DATE=2022-11-18T14:33:25.448251+00:00
> AIRFLOW_CTX_TRY_NUMBER=1
> AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-18T14:33:25.448251+00:00
> [2022-11-18T08:33:38.388-0600] {base.py:71} INFO - Using connection ID 'fivetran_dev' for task execution.
> [2022-11-18T08:33:38.398-0600] {local_task_job.py:164} INFO - Task exited with return code Negsignal.SIGSEGV

I have reached out to FiveTran, and they have told me this is not a Fivetran or Fivetran provider problem and they have suggested reaching out to airflow as this looks like an airflow related issue.

**Comment by Taragolis** [comment]
This is know issue related to MacOS. Try to set environment variable `no_proxy` to `*` e.g.: `export no_proxy='*'`. This might help.

BTW, there is quite a few other discussions related to this issue and nature of SIGSEGV signal (tl;dr issues on OS side, C/C++ libraries): https://github.com/apache/airflow/discussions?discussions_q=Negsignal.SIGSEGV

**Comment by nc-tannerum** [comment]
The export no_proxy='*' did not work for me. I will check other discussions.

**Reply by Taragolis** [reply]
you might also set environment variable `PYTHONFAULTHANDLER` to `true` and check your scheduler/worker after your task failed.
